src/ophyd_async/epics/adpilatus/_pilatus_io.py

```diff
@@ -14,11 +16,7 @@ class PilatusTriggerMode(StrictEnum):
 
 
 class PilatusDriverIO(adcore.ADBaseIO):
-    """Mirrors the interface provided by ADPilatus/db/pilatus.template."""
+    """This mirrors the interface provided by ADPilatus/db/pilatus.template."""
 
```

Comment:
We should link to the HTML docs here:
```suggestion
    """This mirrors the interface provided by ADPilatus/db/pilatus.template."""
    """See HTML docs at https://areadetector.github.io/areaDetector/ADPilatus/pilatusDoc.html"""
```